tools/webpack/blocks.js

```diff
@@ -54,12 +54,12 @@ const createEntrypoints = () => {
 	/*
 	 * Returns an array of paths to block view files within the `@wordpress/block-library` package.
 	 * These paths can be matched by the regex `blockViewRegex` in order to extract
-	 * the block's filename.
+	 * the block's filename. All blocks were migrated to script modules but the Form block.
 	 *
 	 * Returns an empty array if no files were found.
 	 */
 	const blockViewScriptPaths = fastGlob.sync(
-		'./packages/block-library/build-module/**/view*.js'
+		'./packages/block-library/build-module/form/view.js'
 	);
```

Comment:
@gziolo In a follow-up we can make this a `viewScriptModule` and remove this single case. It seems to be a single module with no dependencies, an easy migration.

Reply:
Yes, it seems straightforward but I went with a quick change here to rule it out as a source of issue.